doc/src/general-rules.md

```diff
@@ -69,7 +69,7 @@ Follow the following steps to create a new proposal:
 
 Solutions must be automated using one of the tools previously approved and listed [here](tools.md#approved-tools):
 
-* Any new tool that participants want to enable will require an application using the Issue template [Tool application](todo.md).
+* Any new tool that participants want to enable will require an application using the issue template for [tool applications](https://github.com/model-checking/verify-rust-std/blob/main/.github/TOOL_REQUEST_TEMPLATE.md).
```

Comment:
should we move this file to be inside the book?

Reply:
We have our template for challenges in the book, so it makes sense to keep the tool application one also in the book. So, implemented as suggested.